test/unit/model/execution/DeliveryExecutionListTest.php

```diff
@@ -292,6 +292,7 @@ public function testLastPauseReason(): void
             [
                 DeliveryLog::DATA => [
                     'reason' => ['reason'],
+                    'context' => '/taoProctoring/Monitor/pauseExecutions'
```

Comment:
Maybe it would be better to use `tao_helpers_Uri::uri('pauseExecutions', 'Monitor', 'taoProctoring')` or `_url('pauseExecutions', 'Monitor', 'taoProctoring')` instead?

Reply:
added here: https://github.com/oat-sa/extension-tao-proctoring/pull/836/files#diff-8bee313ee340421126162b2d0d1f6a2cR165